src/EditorFeatures/Test/Workspaces/TestWorkspaceFactory_XmlConsumption.cs

```diff
@@ -225,6 +225,11 @@ public static TestWorkspace CreateWorkspace(string xmlDefinition, bool completed
             if (projectElement.Attribute(FilePathAttributeName) != null)
             {
                 filePath = projectElement.Attribute(FilePathAttributeName).Value;
+                if (string.Compare(filePath, "null", StringComparison.Ordinal) == 0 || string.Compare(filePath, "Nothing", StringComparison.OrdinalIgnoreCase) == 0)
```

Comment:
I'd just pick one magic string to make it easier to grep for. Honestly, make it *really* magic -- have the property called TestWorkspaceFactory.NullPath, and then have the test do <%= TestWorkspaceFactory.NullPath %> in the XML literal.